recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
I guess by 'stop' you mean that we should support custom ExecStop commands for stopping the applications, like eg. pulseaudio could use ExecStop=/usr/bin/pulseaudio -k, right ? If it is given we spit that out into the service file, if it is not given we let systemd stop it with its normal signalling sequence.

However, what would install and uninstall stand for ?

One thing missing from the manifest is the list of services that should be enabled for Ostro-based nspawned system containers, IOW containers that run their own instance of systemd and a full OS userspace. For such containers, if we don't disable or mask the active services on the host, we'll end up running the union of the host services + application-specific services inside the container. We should mask (or disable) all services which are not supposed to be enabled during image baking, or when we start up the service/container.

I'll add the necessary keys for this to the manifest. However, could you elaborate on the install/uninstall idea, and also the stop part if I misunderstood it.